apps/ui/eslint.config.js

```diff
@@ -0,0 +1,28 @@
+import js from '@eslint/js'
+import globals from 'globals'
+import reactHooks from 'eslint-plugin-react-hooks'
+import reactRefresh from 'eslint-plugin-react-refresh'
+import tseslint from 'typescript-eslint'
+
+export default tseslint.config(
+  { ignores: ['dist'] },
+  {
+    extends: [js.configs.recommended, ...tseslint.configs.recommended],
+    files: ['**/*.{ts,tsx}'],
+    languageOptions: {
+      ecmaVersion: 2020,
+      globals: globals.browser,
+    },
+    plugins: {
+      'react-hooks': reactHooks,
+      'react-refresh': reactRefresh,
+    },
+    rules: {
+      ...reactHooks.configs.recommended.rules,
```

Comment:
nothing wrong here, just a heads up - I think this will enforce far stricter hook rules than we're used to. specifically regarding hook dependencies

Reply:
This seems like a good thing to start with, no?

Reply:
@joneubank 100%! as I say, nothing wrong at all, just a heads up. We had to tone down our eslint usage in a previous project (can't recall exactly which one) because we had so many things to fix. 

I agree - very good thing to start with - nice one @demariadaniel !